msal-auth-node/README.md

```diff
@@ -0,0 +1,115 @@
+-->
+
+This Node.js API protects its own protected endpoint using JWT scope validation.
+
+```console
+$ curl http://localhost:8080/ -H "Authorization: Bearer {valid-access-token}"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Node.js 16+
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Configure an application to expose a web API](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis) to register the sample API and expose its scopes.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `Node Web API`                                                               | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                 |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                               |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample with app registration values
+
+```javascript
+auth: {
+  // 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+  tenant: '',
+
+  // 'Application (client) ID' of app registration in the Azure portal - this value is a GUID
+  audience: ''
+}
+```
+
+### 3. Install package(s)
+
+To install Node.js libraries:
```

Comment:
Nit: Probably also installs MSAL?

Reply:
Fun fact, no MSAL here. MSAL doesn't expose JWT token validation. In Node.js land, it's used for acquiring tokens, but no validation functions. Whee!

Reply:
@ckittel Got it. @skabou can you take a pass through the README and ensure it reflects the sample's reality? For example:

- https://github.com/Azure-Samples/ms-identity-docs-code-javascript/pull/4/files#diff-0c1cb6767503c01d5864fa962e7693936eb69fea98e06458468d7a38cffa5438R91
- https://github.com/Azure-Samples/ms-identity-docs-code-javascript/pull/4/files#diff-0c1cb6767503c01d5864fa962e7693936eb69fea98e06458468d7a38cffa5438R100

I'm seeing a couple other things, as well, so another review coming shortly.